atala-prism-sdk/src/commonMain/kotlin/io/iohk/atala/prism/walletsdk/prismagent/protocols/ProtocolType.kt

```diff
@@ -21,6 +21,8 @@ enum class ProtocolType(val value: String) {
     DidcommPresentation("https://didcomm.atalaprism.io/present-proof/3.0/presentation"),
     DidcommRequestPresentation("https://didcomm.atalaprism.io/present-proof/3.0/request-presentation"),
     DidcommProposePresentation("https://didcomm.atalaprism.io/present-proof/3.0/propose-presentation"),
+    DidcommPresentationDefinition("https://didcomm.atalaprism.io/present-exchange/3.0/presentation-definition"),
+    DidcommPresentationSubmission("https://didcomm.atalaprism.io/present-exchange/3.0/presentation-submission"),
```

Comment:
I don't think we need this. Presentation definition and submission are not didcomm related, so they should not be in here. They are not messages, they are content of messages.